src/molecules/product-grid-item/ProductGridItem.html

```diff
@@ -0,0 +1,118 @@
+<a-list-item class="a-product-grid-item">
+  <!--@slot Custom product badge (Scoped slot)-->
+  <slot
+    name="badge"
+    :badge-text="badgeText"
+  >
+    <a-badge
+      v-if="badgeText"
+      class="a-product-grid-item__badge"
+    >
+      {{ badgeText }}
+    </a-badge>
+  </slot>
+
+  <!--@slot Custom product image (Scoped slot)-->
+  <slot
+    name="image"
+    :image-url="imageUrl"
+    :alt="name"
+  >
+    <router-link
+      :to="url"
+      class="a-product-grid-item__link"
+    >
+      <a-image
+        :src="imageUrl"
+        :alt="name"
+        class="a-product-grid-item__image"
+      />
+    </router-link>
+  </slot>
+
+  <section class="a-product-grid-item__details">
+    <!--@slot Custom product name (Scoped slot)-->
+    <slot
+      name="name"
+      :product-name="name"
+      :product-url="url"
+    >
+      <h2 class="a-product-grid-item__name">
+        <router-link
+          :to="url"
+          class="a-product-grid-item__link a-product-grid-item__link--name"
+        >
+          {{ name }}
+        </router-link>
+      </h2>
+    </slot>
+
+    <div class="a-product-grid-item__price">
+      <!--@slot Custom product price (Scoped slot)-->
+      <slot
+        name="price"
+        :price="price"
+        :special-price="specialPrice"
+        :old-price="oldPrice"
+      >
+        <a-price
+          :price="price"
+          :special-price="specialPrice"
+          :old-price="oldPrice"
+        />
+      </slot>
+    </div>
+
+    <!--TODO Add rating component-->
+    <div class="a-product-grid-item__options">
+      <!--@slot Custom product options (Named slot)-->
+      <slot name="options" />
+    </div>
+
+    <div class="a-product-grid-item__actions">
+      <form class="a-product-grid-item__primary-form">
+        <!--@slot Custom add to cart action (Scoped slot)-->
+        <slot
+          name="addToCart"
+          :add-to-cart="addToCart"
+          :add-to-cart-text="addToCartText"
+        >
+          <a-button
+            class="button--secondary a-product-grid-item__primary-action"
+            @click.native="addToCart"
+          >
+            {{ addToCartText }}
+          </a-button>
+        </slot>
+      </form>
+
+      <div class="a-product-grid-item__secondary-action">
+        <!--@slot Custom add to wishlist action (Scoped slot)-->
+        <slot
+          name="addToWishList"
+          :add-to-wishlist="addToWishList"
+        >
+          <a-button
+            aria-label="Add to Wish List"
```

Comment:
It should be passed as prop for translation

Reply:
Right, I'll also do the same thing for the compare element - thx!